### PR TITLE
moyo header task

### PR DESCRIPTION
**Pull request by mttony**
[DEMO LINK](https://mttony.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://mttony.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,62 @@
...
+            Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a 
+              href="#smartphones" class="nav__link"
+            >
```
Comment (SanyaBratashchuk) [inline src/index.html:39]:
```suggestion
            <a 
              href="#smartphones" 
              class="nav__link"
            >
```

File: src/style.css [hunk 2]
```diff
@@ -1 +1,70 @@
...
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  color: #000;
+}
+
+.header {
+  height: 60px;
```
Comment (SanyaBratashchuk) [inline src/style.css:12]: do not hardcode heigth, use vertical paddings for linsk

File: src/style.css [hunk 3]
```diff
@@ -1 +1,70 @@
...
+  padding: 23px 0;
+}
+
+.nav__item:not(:last-child) {
+  padding-right: 20px;
+}
+
+.nav__link:hover,
+.is-active {
+  color: #00acdc;
```
Comment (SanyaBratashchuk) [inline src/style.css:58]: move colors to the variables

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 2 above)
Comment (mttony) [reply src/style.css:12]: I've tried that but 1 px somewhere getting lost and it fails the tests.
I've asked about that in chat and was told to set height of header (https://mate-parttime.slack.com/archives/C03UQS59NRY/p1666945212068099)
And that worked, all tests were passed.
Need your help here.

File: src/style.css [hunk 4]
```diff
@@ -1 +1,72 @@
-/* Styles go here */
+* {
+  margin: 0;
+}
```
Comment (DarkMistyRoom) [inline src/style.css:3]: don't apply anything except box-sizing to *

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
Try to set the height of the header by setting it directly to `a`

File: src/style.css [hunk 5]
```diff
@@ -1 +1,74 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
```
Comment (stanislav-topikha) [inline src/style.css:7]: add fallback font

**Review by stanislav-topikha — APPROVED** [review]

File: src/style.css (hunk 5 above)
Comment (mttony) [reply src/style.css]: done!

**Pull request closed without merging**